### PR TITLE
Fix/toasts and qa bugs

### PR DESCRIPTION
**Pull request by Aerilym**

### CODE REVIEW

File: apps/staking/components/DevSheet.tsx [hunk 1]
```diff
@@ -407,3 +408,91 @@ async function ejectNodes({
   console.log(`Total: ${total}`);
   console.log(`Actual: ${idsToBoot.length}`);
 }
+
+const toastWorkshopFallbackText = 'Toast Workshop 🍞';
```
Comment (yougotwill) [inline apps/staking/components/DevSheet.tsx:412]: freshly baked

File: apps/staking/components/DevSheet.tsx [hunk 2]
```diff
@@ -407,3 +408,91 @@ async function ejectNodes({
...
+      <Button
+        data-testid="button:toast-workshop"
+        size="xs"
+        variant="secondary"
+        onClick={() => {
+          toast.handleError(new Error(text));
+        }}
+      >
+        Error
+      </Button>
```
Comment (yougotwill) [inline apps/staking/components/DevSheet.tsx:495]: I assume this doesn't localization?

File: packages/ui/components/Module.tsx [hunk 3]
```diff
@@ -18,17 +18,22 @@ export const outerModuleVariants = cva(
...
       size: 'default',
+      outline: true,
     },
   }
 );
 
 const innerModuleVariants = cva(
   cn(
-    'rounded-2xl w-full h-full flex align-middle flex-col bg-module',
+    'rounded-[15px] w-full h-full flex align-middle flex-col bg-module',
```
Comment (yougotwill) [inline packages/ui/components/Module.tsx:36]: happy with a hardcoded pixel value?

File: apps/staking/components/DevSheet.tsx (hunk 2 above)
Comment (Aerilym) [reply apps/staking/components/DevSheet.tsx:495]: yeah no user facing content in dev sheet

File: packages/ui/components/Module.tsx (hunk 3 above)
Comment (Aerilym) [reply packages/ui/components/Module.tsx:36]: yeah its the inner radius, has to be hard coded to keep a consistent border width

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

**Pull request merged**